runtime/ops/http.rs

```diff
@@ -280,12 +280,13 @@ fn op_http_start(
 
   if let Some(resource_rc) = state
     .resource_table
-    .take::<TlsServerStreamResource>(tcp_stream_rid)
+    .take::<TlsStreamResource>(tcp_stream_rid)
   {
-    let resource = Rc::try_unwrap(resource_rc)
-      .expect("Only a single use of this resource should happen");
+    let resource = Rc::try_unwrap(resource_rc).unwrap_or_else(|_| {
+      panic!("Only a single use of this resource should happen")
+    });
```

Comment:
Does this change have a purpose? It looks like the same thing to me...

Reply:
`Result::<T, E>::unwrap()` requires `E` to implement `Debug`.
It's un-done, I've added `Debug` impls now.